make/targets_list.mk

```diff
@@ -12,6 +12,7 @@ OFFICIAL_TARGETS  = \
     SPRACINGF3NEO \
     SPRACINGF4EVO \
     SPRACINGF7DUAL \
+    SPRACINGH7EXTREME \
```

Comment:
What is this list used for?

Reply:
there's a make file goal to build official targets.